README.md

```diff
@@ -139,7 +139,7 @@ end
 
 #### Import
 
-**Note:  As of now, imports do not trigger callbacks.**
+**Note:  Imports do not trigger callbacks automatically.  See [Running Callbacks](#running-callbacks).**
```

Comment:
Should we mention that it doesn't run `validate` either?

Reply:
As of now it will skip invalid records.

https://github.com/amberframework/granite/blob/master/src/adapter/mysql.cr#L94

Reply:
@Blacksmoke16 This is always a pain with bulk import one how we handle it.  I will merge this since it's not related to the readme (yet).